src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -464,22 +523,30 @@ private SyntaxNode GetNodeContainingTargetNode(SyntaxNode matchingNode)
             return SyntaxFactory.SeparatedList(newArgumentsWithTrivia, GetSeparators(arguments, numSeparatorsToSkip));
         }
 
-        private static List<T> TransferLeadingWhitespaceTrivia<T, U>(IEnumerable<T> newArguments, SeparatedSyntaxList<U> oldArguments)
+        private List<T> TransferLeadingWhitespaceTrivia<T, U>(IEnumerable<T> newArguments, SeparatedSyntaxList<U> oldArguments)
             where T : SyntaxNode
             where U : SyntaxNode
         {
             var result = new List<T>();
```

Comment:
bleagh that this is List.  Can you move to ImmutableArray?

Reply:
Fixed in the next review.